src/main/java/seedu/foodrem/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ITEM_BOUGHT_DATE = new Prefix("bgt/");
     public static final Prefix PREFIX_ITEM_EXPIRY_DATE = new Prefix("exp/");
     public static final Prefix PREFIX_ITEM_PRICE = new Prefix("p/");
+    public static final Prefix PREFIX_ITEM_REMARKS = new Prefix("r/");
+    public static final Prefix PREFIX_ID = new Prefix("id/");
```

Comment:
Why was this added back?

Reply:
Hmm I see that this is because the recently-merged remarks command requires the `id/`. Let's create an issue to standardise this and remove the `id/` prefix from the remarks command in order to match the other commands. @yixiann Can you open the issue?

Reply:
I think it was automatically added back during a merge, will remove. The remarks command do not use id/